app/views/admin/accounting/transactions/_form.slim

```diff
@@ -0,0 +1,45 @@
+= simple_form_for([:admin, @transaction],
+    html: { \
+      class: 'form-horizontal transaction-form',
+    },
+    wrapper: :horizontal_form_condensed,
+    defaults: { \
+      input_html: {class: 'form-element form-control'},
+    },
+  ) do |f|
+
+  = error_notification(f)
+
+  = f.hidden_field :project_id
+
+  = f.input :loan_transaction_type_value
+    .view-element
+      - if @transaction.loan_transaction_type_value
+        = @transaction.loan_transaction_type_value.capitalize
+    = f.input_field :loan_transaction_type_value, collection: @loan_transaction_types
+
+  = f.input :txn_date
+    .view-element = ldate(@transaction.txn_date)
+    = f.input_field :txn_date, as: :date_picker
+
+  = f.input :account
+    .view-element
+      - if @transaction.account
+        = @transaction.account.name
+    = f.input_field :account, collection: @accounts
```

Comment:
The Rails view expects `account` instead of `account_id`, but the parameters for create transaction expect `account_id`. Usually `f.association` would work, but because of a show/edit configuration here, `f.association` has some styling issues. What do you recommend?

Reply:
Let's discuss when we pair.